src/iperf_api.c

```diff
@@ -953,11 +953,15 @@ iperf_on_connect(struct iperf_test *test)
     struct sockaddr_in *sa_inP;
     struct sockaddr_in6 *sa_in6P;
     socklen_t len;
+    unsigned long long now_millisecs;
 
-    now_secs = time((time_t*) 0);
-    (void) strftime(now_str, sizeof(now_str), rfc1123_fmt, gmtime(&now_secs));
+    clock_gettime(CLOCK_REALTIME, &time_spec);
+    // Unix timestamp in milliseconds (local wall clock time)
+    now_millisecs = time_spec.tv_sec * 1000000 + time_spec.tv_nsec / 1000000;
+
+    (void) strftime(now_str, sizeof(now_str), rfc1123_fmt, gmtime(&time_spec.tv_sec));
     if (test->json_output)
-	cJSON_AddItemToObject(test->json_start, "timestamp", iperf_json_printf("time: %s  timesecs: %d", now_str, (int64_t) now_secs));
+	cJSON_AddItemToObject(test->json_start, "timestamp", iperf_json_printf("time: %s  timesecs: %d  timemillisecs: %d", now_str, (int64_t) time_spec.tv_sec, now_millisecs));
```

Comment:
nit: I won't claim I like that new key name. I see it fits with the existing one. Shorter alternatives would be `time_ms` or `timemsecs` ... I'm fine with using the proposed one here.

Reply:
Any preference by the code owners? @bmah888

Reply:
I have a weak preference for `timemsecs`